gsa/src/web/components/dialog/__tests__/closebutton.js

```diff
@@ -0,0 +1,64 @@
+/* Copyright (C) 2018 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import React from 'react';
+
+import {render, fireEvent} from 'web/utils/testing';
+import CloseButton from '../closebutton';
+import {
+  ICON_SIZE_LARGE_PIXELS,
+  ICON_SIZE_MEDIUM_PIXELS,
+} from 'web/components/icon/withIconSize';
+
+describe('Dialog CloseButton tests', () => {
+
+  test('should render', () => {
+    const {element} = render(
+      <CloseButton onClick={() => {}}/>
+    );
+
+    expect(element).toMatchSnapshot();
+    expect(element).toHaveAttribute('title', 'Close');
```

Comment:
Do you need to make sure locale is set to 'en' here? The title will be 'Schliessen' in German for example.